example_configs/zookeeper.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+  - pattern: "org.apache.ZooKeeperService<name0=ReplicatedServer_id(\\d)><>(\\w+)"
+    name: "zookeeper_$2"
+    labels:
+      brokerId: "$1"
```

Comment:
Per #42, this shouldn't be a label from instrumentation as it's really a target label.

By convention we also do `broker_id`.